pkg/controller/signup_test.go

```diff
@@ -526,6 +686,7 @@ type FakeSignupService struct {
 	MockCreateUserSignup func(ctx *gin.Context) (*crtapi.UserSignup, error)
 	MockGetUserSignup    func(userID string) (*crtapi.UserSignup, error)
 	MockUpdateUserSignup func(userSignup *crtapi.UserSignup) (*crtapi.UserSignup, error)
+	//MockUpdateWithVerificationCode func(dailyLimit int, responseBody map[string]string, userID, code string) (*crtapi.UserSignup, error)
```

Comment:
I believe you can remove this commented out line if it's not useful anymore

Reply:
removed here: https://github.com/codeready-toolchain/registration-service/blob/60ca7377aabe427ed8f8e3d642380b1b8396454e/pkg/controller/signup_test.go#L681-L686